launchcontainer/launchcontainer.py

```diff
@@ -14,13 +14,34 @@
 from xblock.fields import Scope, String
 from xblock.fragment import Fragment
 
+try:
+    from openedx.core.djangoapps.site_configuration import helpers as siteconfig_helpers
+except ImportError:  # We're not in an openedx environment.
+    siteconfig_helpers = None
 
-log = logging.getLogger(__name__)
 
+logger = logging.getLogger(__name__)
+DEFAULT_WHARF_URL = 'https://wharf.appsembler.com/isc/newdeploy'
```

Comment:
Could we get rid of this default URL? Someone can e.g. install this XBlock and forget to update the URL. And even we won't use this default in most cases.